barretenberg/cpp/pil/avm/gadgets/cmp.pil

```diff
@@ -16,6 +16,26 @@ namespace cmp(256);
     pol commit input_b;
     pol commit result;
 
+    // ========= NON FF GT Short-circuit computation ===============================
+    // If this is a non-ff check, we can short circuit (a > b) by checking
+    // 0 < a - b < 2**128 --> i.e. we just check we dont underlow which for "small" sized 128-bit number is just a 
```

Comment:
Since later we use `a -b -1`, we might directly mention that we check that:
`0 <= a-b-1 < 2**128` which is exactly a 128-bit range check.